site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -132,14 +132,40 @@ public function generateHistogramData($overall_scores) {
         }
         return $histogram;
     }
-
+    /**
+     * function to filter out students in the same team as Grader
+     * @param Gradeable $gradeable
+     * @param string $grader
+     * @param Array $students
+     * @return Array $filtered_grading_info
+     */
+    private function filterTeamStudents(Gradeable $gradeable, string $grader, array $students) {
```

Comment:
This should be a resonable method but what if the following condition happens?
A team with 2 members are assigned to grad for each other and the `number_to_grade` equals to 1. After implementing the fileterTeamStudents() function, this team will not be graded by anyone. 
So how about adding a notice for those team assignments which are not properly assigned?

Reply:
sure! @Xiayucheng1212 
In this particular case, I can inform user about the issue and suggest on using  "Edit Students" to add a new student who isn't a team member for that grader
i hope any of the maintainers to give me suggestions on where to notify the user as we have lot of ways (using alert, on the peer matrix, UI, pop-up and etc.)

Reply:
Lets use alert for now.